model/websocket_message.go

```diff
@@ -92,7 +97,8 @@ type WebsocketBroadcast struct {
 	ContainsSensitiveData bool            `json:"-"`
 	// ReliableClusterSend indicates whether or not the message should
 	// be sent through the cluster using the reliable, TCP backed channel.
-	ReliableClusterSend bool `json:"-"`
+	ReliableClusterSend bool               `json:"-"`
+	SubjectID           WebsocketSubjectID `json:"-"`
```

Comment:
Curious to know the reason behind not showing the result in the json output. This will prevent it from being sent in cluster messages so the subject will be entirely lost when broadcasted to other nodes.

Reply:
Ok, I'm totally unfamiliar with that cluster sending behaviour. My purpose was to avoid responding with that new field in the WebSocket message response payload since clients don't need to be aware of the subject ID associated to messages. Seems these two concerns are coupled?